config.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+
+  vip:
+    description: |
+      Virtual IP to use to front mysql router units. Used only in case of external node connection.
+    type: string
```

Comment:
cc: @delgod and @7annaba3l , we are syncing the option with pgbouncer (which got it from legacy pgbouncer).  Are you OK for common UX here?

Reply:
@taurus-forever : Yes. Generally speaking we should always favor common UX between PostgreSQL and MySQL.